readthedocs/projects/models.py

```diff
@@ -172,7 +172,12 @@ class AddonsConfig(TimeStampedModel):
     doc_diff_enabled = models.BooleanField(default=True)
     doc_diff_show_additions = models.BooleanField(default=True)
     doc_diff_show_deletions = models.BooleanField(default=True)
-    doc_diff_root_selector = models.CharField(null=True, blank=True, max_length=128)
+    doc_diff_root_selector = models.CharField(
+        null=True,
+        blank=True,
+        max_length=128,
+        help_text="CSS selector for the main content of the page",
```

Comment:
Should we note the default here?

Reply:
I'm fine with that. However, I'm not putting too much data in `help_text` for now because I understand we want to move away from that based on some Anthony's comments in other PRs related to the admin UI for the addons.

Ideally, this should be explained better with more UI elements and documentation (see https://github.com/readthedocs/ext-theme/issues/293)

I used a placeholder here to show the default value 6da1bb0c4. I think it's pretty clear now:

![Screenshot_2024-03-05_09-45-03](https://github.com/readthedocs/readthedocs.org/assets/244656/38c48ec2-d493-4be1-a035-478fef92d292)

Thanks for the suggestion 👍🏼